### PR TITLE
[PHB] Typo Fixes

### PR DESCRIPTION
**Pull request by sammy6345**
Typo fixes for Criminal and Urban Bounty Hunter backgrounds and in the Asleep spell description.

### CODE REVIEW

File: core/players-handbook/spells.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <elements>
 	<info>
 		<name>Spells</name>
-		<update version="0.4.9">
+		<update version="0.4.10">
```
Comment (NickVendel) [inline core/players-handbook/spells.xml:5]: we usually change it like this, when going from 9 to 10
```suggestion
		<update version="0.5.0">
```

**Comment by NickVendel** [comment]
Also change title to `[MISC]`, since you make updates to multiple sources (PHB and SCAG)

**Comment by sammy6345** [comment]
Ok, will do those next time. Thanks for the feedback.

**Review by kgirtxd — CHANGES_REQUESTED** [review]
Once you commit Nick's suggestion Id say this pull is ready for merge

**Review by kgirtxd — APPROVED** [review]

**Review by NickVendel — APPROVED** [review]

**Review by AngryGuineaPig — APPROVED** [review]

**Pull request merged**